service/implementation/auto_request_api/sport_data_managers/teams_data_manager.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from database.models import Sport
+from database.session import SessionLocal
```

Comment:
we shouldn't use database layer directly in BusinessLogic. I mean we can't expose Session or Database Models (tables) to our BusinessLayer since it violates SOLID. 

Data Access Layer should be separate and only expose its functionality through interfaces (public classes in our case. Some DALs. (like the ones Vika has made)).